chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -263,12 +263,19 @@ const setCustomContextParams = (props?: ILiveChatWidgetProps) => {
     if (!isUndefinedOrEmpty(persistedState?.domainStates?.customContext)) {
         TelemetryHelper.logLoadingEvent(LogLevel.INFO, {
             Event: TelemetryEvent.SetCustomContext,
-            Description: "Setting custom context for unauthenicated chat"
+            Description: "Setting custom context for unauthenticated chat"
         });
 
         optionalParams = Object.assign({}, optionalParams, {
             customContext: persistedState?.domainStates?.customContext
         });
+    } else {
```

Comment:
Please verify if non-popout still work with this change

Reply:
It is working. I have attached test evidences for Auth/UnAuth with embedded & PopOut mode